pkg/test/env/factory_wiremock.go

```diff
@@ -0,0 +1,97 @@
+package env
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/applike/gosoline/pkg/cfg"
+	"github.com/applike/gosoline/pkg/mon"
+	"io/ioutil"
+	"net/http"
+	"path/filepath"
+)
+
+func init() {
+	componentFactories[componentWiremock] = new(wiremockFactory)
+}
+
+const componentWiremock = "wiremock"
+
+type wiremockSettings struct {
+	ComponentBaseSettings
+	ComponentContainerSettings
+	Mocks string `cfg:"mocks"`
+	Port  int    `cfg:"port" default:"0"`
+}
+
+type wiremockFactory struct {
+}
+
+func (f *wiremockFactory) Detect(_ cfg.Config, _ *ComponentsConfigManager) error {
+	return nil
+}
+
+func (f *wiremockFactory) GetSettingsSchema() ComponentBaseSettingsAware {
+	return &wiremockSettings{}
+}
+
+func (f *wiremockFactory) ConfigureContainer(settings interface{}) *containerConfig {
+	s := settings.(*wiremockSettings)
+
+	return &containerConfig{
+		Repository: "rodolpheche/wiremock",
+		Tag:        "2.26.3-alpine",
+		PortBindings: portBindings{
+			"8080/tcp": s.Port,
+		},
+		ExpireAfter: s.ExpireAfter,
+	}
+}
+
+func (f *wiremockFactory) HealthCheck(_ interface{}) ComponentHealthCheck {
+	return func(container *container) error {
+		binding := container.bindings["8080/tcp"]
+		url := f.getUrl(binding)
+
+		resp, err := http.Get(url)
```

Comment:
For me, `/__admin` (which we get from `getUrl`) redirects to `/__admin/`, only then do I get a 200 response. Not quite sure how much we need to check the state of wiremock itself, but it sounds like it would be better to get a 200 response than a redirect.